tox.ini

```
@@ -0,0 +1,26 @@
+[tox]
+envlist = py3{8,9,10,11,12}, mypy, black, pylint
+isolated_build = True
+skip_missing_interpreters = True
+
+[testenv]
+deps = pytest
+commands = pytest
+
+[testenv:black]
+deps = black
+skip_install = True
+commands = black --check --diff sphinxlint/ tests/
```

Do we need the dir names? 
```suggestion
commands = black --check --diff
```

---

Yes:

```
$ black --check --diff
Usage: black [OPTIONS] SRC ...

One of 'SRC' or 'code' is required.
```

Oh maybe we could use `black --check --diff .`.